### PR TITLE
chore: change component example generation so its not using deprecate…

### PR DESCRIPTION
**Pull request by Brailor**
…d props anymore

### CODE REVIEW

**Review by ke1k0 — DISMISSED** [review]
Looks good! Shall we make a ticket for making a cleanup version of this for next, after this is merged?

File: packages/ui-list/src/List/index.js [hunk 1]
```diff
@@ -50,7 +50,9 @@ category: components
 ---
 **/
 @deprecated('8.0.0', {
-  variant: 'List with the isUnstyled boolean or InlineList'
+  variant: 'List with the isUnstyled boolean or InlineList',
+  delimeter:
+    'with delimiter set to [pipe, slash, arrow] will only be available when using [InlineList] as of version 8.0.0.'
```
Comment (matyasf) [inline packages/ui-list/src/List/index.js:55]: This should not be here. Also there is a type "delimeter"

File: packages/ui-list/src/List/index.js [hunk 2]
```diff
@@ -64,12 +66,14 @@ class List extends Component {
...
-      ({ propValue }) =>
-        `with 'delimiter' set to ${propValue} will only be available when using [InlineList] as of version 8.0.0.`
-    ),
+    delimiter: PropTypes.oneOf([
+      'none',
+      'dashed',
+      'solid',
+      'pipe',
+      'slash',
+      'arrow'
```
Comment (matyasf) [inline packages/ui-list/src/List/index.js:75]: these are removed

File: packages/ui-view/src/View/index.js [hunk 3]
```diff
@@ -57,7 +57,12 @@ category: components
...
 @deprecated('8.0.0', {
   focused: 'withFocusOutline',
-  visualDebug: 'withVisualDebug'
+  visualDebug: 'withVisualDebug',
+  background: `In version 8.0.0,
+    the value <default> for background will be changed to <primary>,
+    the value <light> for background will be changed to <secondary>,
+    the value <inverse> for background will be changed to <primary-inverse>.
+    Use these values instead.`
 })
```
Comment (matyasf) [inline packages/ui-view/src/View/index.js:66]: is this OK? If we put in here a deprecation notice, it will change in 9.0

Comment (ke1k0) [reply packages/ui-view/src/View/index.js:66]: After this is merged into master, and master is again merged into next, we need to make cleanup on next to reverse these.

**Review by matyasf — APPROVED** [review]

**Review by ke1k0 — APPROVED** [review]

**Pull request merged**